### PR TITLE
Adding missing lib for building tests with Ninja from command prompt

### PR DESCRIPTION
**Pull request by vhvb1989**
fixes: #1228

### CODE REVIEW

**Comment by vhvb1989** [comment]
Since I am going on vacation. This can be merged by the Storage team if they want to and approve.

File: sdk/storage/azure-storage-common/CMakeLists.txt [hunk 1]
```diff
@@ -81,7 +81,7 @@ target_link_libraries(azure-storage-common PUBLIC Azure::azure-core)
 target_link_libraries(azure-storage-common INTERFACE Threads::Threads ${LIBXML2_LIBRARIES})
 
 if(MSVC)
-    target_link_libraries(azure-storage-common PRIVATE bcrypt)
+    target_link_libraries(azure-storage-common PRIVATE bcrypt Crypt32.lib)
```
Comment (Jinming-Hu) [inline sdk/storage/azure-storage-common/CMakeLists.txt:84]: I think I wrongly changed the KEYWORDS here in PR #1212.

I need to investigate more before approving this PR.

Comment (Jinming-Hu) [inline sdk/storage/azure-storage-common/CMakeLists.txt:84]:
```suggestion
    target_link_libraries(azure-storage-common PRIVATE bcrypt crypt32)
```

**Comment by Jinming-Hu** [comment]
@vhvb1989 Thanks for fixing the issue!

**Review by Jinming-Hu — APPROVED** [review]

**Pull request merged**